spec/factories/protocols.rb

```diff
@@ -3,5 +3,61 @@
     id { SecureRandom.uuid }
     name { Faker::Address.state + " Protocol" }
     follow_up_days { rand(1..60) }
+
+    trait :with_tracked_drugs do
+      protocol_drugs {
+        [
+          build(:protocol_drug,
+            name: "Amlodipine",
+            dosage: "5 mg",
+            rxnorm_code: "329528",
+            stock_tracked: true,
+            drug_category: "hypertension_ccb",
+            protocol_id: id),
+          build(:protocol_drug,
+            name: "Amlodipine",
+            dosage: "10 mg",
+            rxnorm_code: "329526",
+            stock_tracked: true,
+            drug_category: "hypertension_ccb",
+            protocol_id: id),
+          build(:protocol_drug,
+            name: "Telmisartan",
+            dosage: "40 mg",
+            rxnorm_code: "316764",
+            stock_tracked: true,
+            drug_category: "hypertension_arb",
+            protocol_id: id),
+          build(:protocol_drug,
+            name: "Telmisartan",
+            dosage: "80 mg",
+            rxnorm_code: "316765",
+            stock_tracked: true,
+            drug_category: "hypertension_arb",
+            protocol_id: id),
+          build(:protocol_drug,
+            name: "Losartan",
+            dosage: "50 mg",
+            rxnorm_code: "979467",
+            stock_tracked: true,
+            drug_category: "hypertension_arb",
+            protocol_id: id),
+          build(:protocol_drug,
+            name: "Hydrochlorthiazide",
+            dosage: "25 mg",
+            rxnorm_code: "316049",
+            stock_tracked: true,
+            drug_category: "hypertension_diuretic",
+            protocol_id: id),
+          build(:protocol_drug,
+            name: "Chlorthalidone",
+            dosage: "12.5 mg",
+            rxnorm_code: "331132",
+            stock_tracked: true,
+            drug_category: "hypertension_diuretic",
+            protocol_id: id)
+        ]
+      }
+    end
```

Comment:
This is very nice. If you're feeling up to it, wanna replace the code in `ProtocolSeeder` with a call to this factory? Optional.

Reply:
Replaced in a90cc9c99ae7bd755b20f4d5e432135bcb13a058